src/main/java/org/janelia/saalfeldlab/n5/universe/metadata/MetadataUtils.java

```diff
@@ -191,21 +192,24 @@ public static String normalizeGroupPath(final String path) {
 		return path;
 	}
 
-	public static String relativePath( final String parent, final String child )
-	{
-		try
-		{
-			final N5URI purl = new N5URI( "?" + parent );
-			final N5URI curl = new N5URI( "?" + child );
-			final Path ppath = Paths.get( purl.normalizeGroupPath());
-			final Path cpath = Paths.get( curl.normalizeGroupPath());
-			return ppath.relativize(cpath).toString();
-		}
-		catch ( final URISyntaxException e )
-		{
-			e.printStackTrace();
-		}
-		return null;
+	/**
+	 * Returns a relative group path from the child absolute path group path child 
+	 * the parent absolute group path.
```

Comment:
@bogovicj this text does not really seem to make sense.

Reply:
Maybe: `Given an absolute child path and a parent path, it returns the child path relativized with respect to the parent path. If the parent is in fact not a parent of the child, the child path is return as is.`